sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -98,6 +100,133 @@ public EditContactSteps(WebDriverHelpers webDriverHelpers) {
           softly.assertAll();
         });
 
+    When(
+        "I check the edited data is correctly displayed on Edit Contact page after editing",
+        () -> {
+          aContact = collectContactDataAfterEdit();
+
+          SoftAssertions softly = new SoftAssertions();
+          softly
+              .assertThat(aContact.getClassification())
+              .isEqualToIgnoringCase(editedContact.getClassification());
+          softly.assertThat(aContact.getMultiDay()).isEqualToIgnoringCase("on");
```

Comment:
Please change `multiDay` into a boolean type because it is a checked or not checked option, and compare only objects not object vs value.

Reply:
reworked. added check as text for checbox's label.